app/konnect/dev-portal/dev-reg.md

```diff
@@ -24,12 +20,25 @@ All Developers must register through the {{site.konnect_short_name}} [Dev Portal
 
    You should receive an email to confirm your email address. Click the verification link in the email.
 
-5. Set a password for your account.
+4. Set a password for your account.
 
 If your admin has auto-approve enabled, you can log into the Dev Portal immediately after setting a password. Learn how to [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
 
 If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. Learn how to [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg).
 
+## Login/Register with SSO {#sso}
+{:.badge .enterprise}
+{{site.konnect_short_name}} Dev Portal supports single sign-on login and registration. This feature must be configured by the {{site.konnect_short_name}} site admin before being made available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
```

Comment:
```suggestion
The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} Admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
```
Here "single sign-on" isn't capped, but you capped it before. Looks like Google has it in lowercase.